components/PicklistBase/PicklistBase.vue

```diff
@@ -36,7 +37,8 @@
                 variant="primary"
                 v-on:click="handleUnitsButton()"
               >
-                {{ units }}
+                <span v-if="allPicked">🚫 Clear {{ units }}</span>
```

Comment:
Leave the text here as just `{{ units }}` for the same reason as expressed above.